Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -529,14 +549,36 @@ class BocList
 
   private static FixValidationErrorOverflowSize(scrollableContainer: HTMLElement)
   {
-    // Sets the width of the validation error field to be as wide as the visible part of the bocList.
+    // Sets the width of the validation error field to be as wide as the visible part of the bocList
+    // minus how much space the td on the left requires to align with the error indicator column.
     // This enables sticky behaviour while still allowing for a scrollbar if required.
 
-    const width = LayoutUtility.GetWidth(scrollableContainer);
+    const availableWidthInContainer = scrollableContainer.clientWidth;
+
+    const td: Nullable<HTMLElement> = scrollableContainer.querySelector('tr.bocListValidationRow td:first-child:not(.bocListValidationFailureCell)');
+
+    let widthBeforeValidationFailureColumn = 0;
+    // width does not matter if the first td is already the one with the error messages (and therefore has child nodes)
```

Comment:
```suggestion
```
the part in the parenthesis is obsolete now and I think we can drop the entire comment because the code says it all.

Reply:
I dont understand which part in the parenthesis youre talking about here. The "hasChildNodes()" part?

Reply:
I was referrring to the selected comment line and this text part of the comment: `(and therefore has child nodes)`
Unless we drop the entire comment.